src/blas/backends/cublas/cublas_level1.cpp

```diff
@@ -65,7 +65,7 @@ inline void asum(Func func, cl::sycl::queue &queue, int64_t n, cl::sycl::buffer<
 #define ASUM_LAUNCHER(TYPE1, TYPE2, CUBLAS_ROUTINE)                             \
     void asum(cl::sycl::queue &queue, int64_t n, cl::sycl::buffer<TYPE1, 1> &x, \
               const int64_t incx, cl::sycl::buffer<TYPE2, 1> &result) {         \
-        asum(CUBLAS_ROUTINE, queue, n, x, incx, result);                        \
+        asum(#CUBLAS_ROUTINE, CUBLAS_ROUTINE, queue, n, x, incx, result);       \
```

Comment:
Again, I think this change is unnecessary if we pass a literal string to the macro.